migrations_scripts/1_deploy_registry_and_add_adapters.js

```diff
@@ -151,6 +153,9 @@ const iETHAddress = '0xA9859874e1743A32409f75bB11549892138BBA1E';
 
 const wethAddress = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2';
 
+const melonAddress = '0xec67005c4e498ec7f55e092bd1d35cbc47c91892';
+const mlnfAddress = '0xd485c308df50f13cb1ec06d5fd3a44a0ae49c7fc';
```

Comment:
This is the MLNF (Melon Fund) token of a specific fund, not globally for the network. The MLNF token is the Shares contract of a fund and one instance of this contract is deployed per fund. E.g. this one belongs to the fund with the address `0x37d5c61240d163d1684cd66e052a81be596dc6ff`.

Reply:
Thanks @fubhy, I will change this.

Reply:
fixed it here 4ead7a6